docs/error-handling.md

```diff
@@ -110,6 +110,15 @@ int git_repository_open(git_repository **repository, const char *path)
 }
 ~~~
 
+Note that some error codes have been defined with a specific meaning in the
+context of callbacks:
+  - `GIT_EUSER` provides a way to bubble up a non libgit2-related failure, which
+  allows it to be preserved all the way up to the initial function call (a `git_cred`
+  setup trying to access an unavailable LDAP server for instance).
+  - `GIT_EPASSTHROUGH` provides a way to tell libgit2 that it should behave as if
+  no callback was provided. This is of special interest to bindings, which would
+  always provide a C function as a "trampoline", and decide at runtime what to do.
```

Comment:
I guess the paragraphs should be aligned with the first character following the bullet point

Reply:
It doesn't matter to the online GitHub display. I'll remove the indents before the `-` though, for consistency.